src/relay/environment.js

```diff
@@ -1,26 +1,31 @@
 // @noflow
 
 import { Environment, Network, RecordSource, Store } from 'relay-runtime';
+import { getSessionToken } from '../helpers/Auth';
 
 require('isomorphic-fetch');
 
-// used when smart FAQ installed as dependency
-const uri = 'https://graphql.kiwi.com';
```

Comment:
this line shoudn't be removed because our env vars are not working properly on kiwi frontend when we install smartfaq as a dependency.